### PR TITLE
Refactor terraform code to meet codex requirements.

### PR DESCRIPTION
**Pull request by Pumpkin-3906**
## Issue number

#000

## Checklist before requesting a review
- [x] The pr title can be used to describe what this pr did in `CHANGELOG.md` file
- [x] I have executed pre-commit on my machine
- [x] I have passed pr-check on my machine

Thanks for your cooperation!

### CODE REVIEW

File: modules/ActiveDirectory/output.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+output "network_security_group_id" {
```
Comment (jiaweitao001) [inline modules/ActiveDirectory/output.tf:1]: According to the codex, we would better name this file `outputs.tf`?

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
It seems like we need rebase this branch from the latest master branch.

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Hey @Pumpkin-3906, it looks like you've merged your `tflint` branch with the `master` branch of your own forked repo, which is outdated, please sync the `master` branch of your own forked repo with upstream first, then try to merge or rebase again, thanks.

**Review by lonegunmanb — APPROVED** [review]
Thanks @Pumpkin-3906, LGTM 🚀

**Pull request merged**